app/controllers/mfa_confirmation_controller.rb

```diff
@@ -43,6 +43,6 @@ def handle_invalid_password
   def handle_max_password_attempts_reached
     analytics.track_event(Analytics::PASSWORD_MAX_ATTEMPTS)
     sign_out
-    redirect_to root_url, alert: t('errors.max_password_attempts_reached')
+    redirect_to root_url, flash: { error: t('errors.max_password_attempts_reached') }
```

Comment:
TIL `redirect_to` can also set the flash!

Reply:
Thankfully the tests caught it, since I too wasn't aware of this usage. 😅